packages/admin/admin-generator/src/commands/generate/generateForm/__tests__/staticSelect.ts

```diff
@@ -0,0 +1,278 @@
+import { ESLint } from "eslint";
+import { promises as fs } from "fs";
+import { buildSchema, introspectionFromSchema } from "graphql";
+import { type JsxAttribute, Project, SyntaxKind } from "ts-morph";
```

Comment:
Did you consider writing the tests with the TS Compiler API as well? Or should we use ts-morph because of its higher-level API?

Reply:
using ts compiler api in tests, would require much more code to write. for example there is no `getDescendantsOfKind`, you would have to do that manually - or write a helper.

The only argument for the lower level tool would be performance, but I consider less code much more important for unit tests (than performance)